src/components/Pages/SetupPage/SetupPage.tsx

```diff
@@ -321,7 +341,7 @@ export const SetupPage = () => {
                 </div>
                 <div className={styles.continue}>
                   <Button primary fluid onClick={handleContinue}>
-                    continue
+                    Continue
```

Comment:
These are the ones fro the randomization page. They don't matter. Just capitalized it for irrelevant reasons as buttons show text in all caps.